src/Root.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  HashRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from 'react-router-dom';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { App } from './App';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="home" element={<Navigate to="/" replace />} />
+          <Route path="people">
+            <Route index element={<PeoplePage />} />
+            <Route path=":slug" element={<PeoplePage />} />
```

Comment:
You have used the same `PeoplePage` component for both the index route and the `:slug` route under the `people` path. If `PeoplePage` is intended to display a list of people, it may not be suitable for displaying a single person's details based on the `slug`. Consider creating a separate `PersonPage` component for the `:slug` route.